custom/icds_reports/models/aggregate.py

```diff
@@ -200,7 +200,7 @@ class AwcLocation(models.Model, AggregateMixin):
     district_id = models.TextField(db_index=True)
     district_name = models.TextField(blank=True, null=True)
     district_site_code = models.TextField(blank=True, null=True)
-    state_id = models.TextField(db_index=True)
+    state_id = models.TextField()
```

Comment:
doesn't this require a django migration?

Reply:
I thought it would too, but when I ran makemigrations yesterday, it didn't show anything related to this. I think its because its an index on an unmanaged table so django doesn't actually keep any state on it. its just for documentation (though I didn't verify this)